index.html

```diff
@@ -0,0 +1,177 @@
+
+        </ul>
+        <div class="header-button"><button type="button" class="button-header">G Gathering</button></div>
+
+      </div>
+    </nav>
+  </section>
+  <div id="mySidenav" class="sidenav">
+    <a href="#" class="closebtn">&times;</a>
+    <a href="about.html">About</a>
+    <a href="#">Program</a>
+    <a href="#">Join</a>
+    <a href="#">Sponsers</a>
+    <a href="#">News</a>
+  </div>
+
+  <section class="home" id="home">
+    <div class="main">
+      <div class="hamburger"><img src="./images/Ham.png" alt="hamburger"></div>
+      <div class="content">
+        <h1>"Welcome! to the Marketing world"</h1>
+        <h2>A GLOBAL GATHERING <br>OF BRAND BUILDERS<br>& MARKETERS</h2>
+        <div class="inner-content">
+          <p>The Marketing Conference spotlights the latest innovations and breakthroughs in the marketing and advertising industry. Connect with the global community and join thought leaders and storytellers as they explore visionary ideas that engage and
+            inspire
+          </p>
+        </div>
+        <div class="inner-content-bottom">
+          <h3>2022.03.17(THU) ~ 19(SAT)</h3>
+          <p>@ The Mirage, Las Vegas, USA</p>
+        </div>
+      </div>
+
+    </div>
+
+
+    <div class="main-program">
+      <div>
+        <h4>Who Attends?</h4>
+        <div class="under-line"></div>
+      </div>
+      <div class="program">
+        <div class="program-type">
+          <img src="./images/Lecture.png" alt="lecture">
+          <h5>Leaders</h5>
+          <p>Expects rounding up strategies and success secrets of the world’s leading brands to get the midst of industry-relevant discussions.
+          </p>
+        </div>
+        <div class="program-type">
+          <img src="./images/Exibition.png" alt="Exhibition">
+          <h5>Artistics</h5>
+          <p>Notion and products promotion before a global audience, know competitor's adequates, to explore the latest innovation</p>
+        </div>
+        <div class="program-type">
+          <img src="./images/forum.png" alt="forum">
+          <h5>Coherent</h5>
+          <p>Learn how to jot epic content, not only garner millions of views but also leave an impact on your target audience.</p>
+        </div>
+        <div class="program-type">
+          <img src="./images/workshop.png" alt="workshop">
+          <h5>Scholars</h5>
+          <p>Drive huge consumer engagement & know about the latest technologies and lead generation strategies</p>
+        </div>
+        <div class="program-type">
+          <img src="./images/Ignite.png" alt="Ignite">
+          <h5>Cognitive</h5>
+          <p>Omnichannel advertisers sought-after names for their creative acumen and tactics will brainstorm solutions.</p>
+        </div>
+      </div>
+      <div class="join-button"><button>Join Global Gathering 2022</button></div>
+      <div class="program-check"><a href="#">SEE THE WHOLE PROGRAM</a>
+      </div>
+    </div>
```

Comment:
Kindly try to match the `main program` section with the `Behance` design in the desktop version width `769px` on the index page, Please make the boxes the same height.
| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/157549826-45a317b0-1ce8-47e2-9f49-9e6c3f3ff11d.png) | ![image](https://user-images.githubusercontent.com/24703171/157549995-ecc93490-f82d-4359-a8f0-05c82d523160.png) |
- [OPTIONAL] Try to add hover effect for the boxes.